internal/elasticsearch/v7/bulk_request_metadata.go

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2022 Meroxa, Inc. and Miquido
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v7
```

Comment:
Is there a difference between the requests (and responses) across different versions, especially v5-v7? 

If there are no changes, it looks like we can take advantage of that, and simplify the client code. v8 is the one being actively maintained, so if v5-v7 are the same, they will stay the same.

Reply:
They are different, unfortunately :/ v6 introduces `RetryOnConflict int` for update action and v7 removes support for indices' types.

Reply:
Too bad!